res/skins/Shade/style.qss

```diff
@@ -83,34 +84,273 @@ WEffectSelector::checked
+  QTreeView::branch:!has-children:!has-siblings:closed:selected,
+  QTreeView::branch:closed:!has-children:has-siblings:selected,
+  QTreeView::branch:open:!has-children:!has-siblings:selected,
+  QTreeView::branch:open:!has-children:has-siblings:selected {
+    border-image: none;
+    background-color: #0f0f0f;
+  }
+
+QTreeView::item:selected {
+  border-image: none;
+  background-image: none;
+  background-color: #333;
+  color: #d2d2d2;
+  }
+
+QTreeView::item:!selected {
+  border-image: none;
+  background-image: none;
+  background-color: #0f0f0f;
+  color: #999;
+  }
+
+  /* BPM lock icon in the library "BPM" column. */
+  #LibraryBPMButton::indicator:checked {
+    image: url(:/images/library/ic_library_checked.png);
+    }
+    #LibraryBPMButton::indicator:unchecked {
+      image: url(:/images/library/ic_library_unchecked.png);
+      }
+  #LibraryBPMButton::item {
+    color: #cfcfcf;
+    }
+    #LibraryBPMButton::item:selected {
+      color: #ededed;
+      background-color: #666;
+      }
+
+	#LibraryBPMSpinBox {
+	  color: #fff;
+	  selection-color: #0f0f0f;
+	  background-color: #0f0f0f;
+	  selection-background-color: #888;
+	  /* This combination of negative top/bottom padding & margin
+		creates a 1px border around the spinbox' text area.
+		Increased left/right padding/margin removes the overlap of
+		text of activated spinbox and left-over text from regular
+		BPM value display. */
+	  padding: -1px -8px -1px -4px;
+	  margin: -2px 0px -2px -2px;
+	  /* As soon as any border property is set, the up/down buttons
+		lose their style & icons
+	  border: 0px;
+	  border-radius: 2px; */
+	}
+
+  #LibraryTimesPlayed::item {
+    color: #cfcfcf;
+    }
+
+	/* Button in library "Preview" column */
+ 	#LibraryPreviewButton {
+		width: 23px;
+		height: 12px;
+		background: transparent;
+		margin: 0px;
+		padding: 0px;
+		border: 1px solid transparent;
+	}
+	#LibraryPreviewButton:!checked{
+		image: url(:/images/library/ic_library_preview_play.png);
+	}
+	#LibraryPreviewButton:!checked:hover {
+    background: #454545;
+	}
+	#LibraryPreviewButton:checked{
+		image: url(:/images/library/ic_library_preview_pause.png);
+		background-color: #0f0f0f;
+		border: 1px solid #333;
+	}
+	#LibraryPreviewButton:checked:hover {
+	  	border: 1px solid #888;
+	}
+
+
+  /* checkbox in library "Played" column */
+  QTableView::indicator {
+    width: 12px;
+    height: 12px;
+    margin: 0px;
+    padding: 0px;
+    color: #cfcfcf;
+    }
+    QTableView::indicator:checked {
+      background: transparent url(skin:/style/style_checkbox_checked.png) no-repeat center center;
+      color: #cfcfcf;
+      }
+    QTableView::indicator:unchecked {
+      background: transparent url(skin:/style/style_checkbox_unchecked.png) no-repeat center center;
+      color: #cfcfcf;
+      }
+  /* selected table row */
+  QTableView::item:selected {
+    color: #fff;
+    background-color: #666;
+  }
+
+/* library search text box */
+WSearchLineEdit {
```

Comment:
`margin-top: 2px;` should be restored here so the searchbox doesn't stick to top part of skin

Reply:
done